Code/Tools/AzTestRunner/src/main.cpp

```diff
@@ -226,6 +226,12 @@ namespace AzTestRunner
             testMainFunction.reset();
         }
 
+        // Construct a retry command if the test fails
+        if (result != 0)
+        {
+            std::cout << "Retry command: " << std::string(argv[0]) << " " << lib << " " << symbol << std::endl;
```

Comment:
Can you provide an example of this output, and how it differs from the existing output on line 121? I don't believe it will provide CWD nor the AzTestRunner executable, thus the user cannot actually paste the "rerun" command back into the terminal.

Also recommend placing a retry command on its own new line, so the user can more easily copy-paste it.

Reply:
Direct output:

`Retry command: C:/Users/evanchia/github/o3de/windows_vs2019/bin/profile/AzTestRunner.exe C:/Users/evanchia/github/o3de/windows_vs2019/bin/profile/AzToolsFramework.Tests.dll AzRunUnitTests`

Reply:
Looks great! Though i'd still be happy with an extra newline to make it easier to copy :)

Reply:
added newline